core/src/test/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43Test.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.applicationinsights.internal.channel.common;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public final class ApacheSender43Test {
+    @Test
+    public void testHttpClientType() {
+        HttpClient tested = new ApacheSender42().getHttpClient();
```

Comment:
According to the class name you should create an `ApacheSender43` here and not 42